app/src/org/commcare/utils/EncryptionUtils.java

```diff
@@ -597,4 +597,19 @@ private static boolean writeSubmissionManifest(
 
         return true;
     }
+
+    public static String getMD5HashAsString(String plainText) {
```

Comment:
It might be a bit annoying for us to replicate this exact process in another environment, can we maybe use the Base64 utility to convert the hash byte[] to a base64 string? That's super easy to replicate in, say, Python with common libs.